### PR TITLE
Js pokecard display & overall styling

### PR DESCRIPTION
**Pull request by jordy1611**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Testing - building tests
- [x] Formatting - no new features
- [ ] Styling- no new features
### Detailed Description (Why is this change required? What problem does it solve? Any challenges that arose while implementing this feature? If so, how were the addressed?)
User can now view their team on the main page
Navigation is only done through the header now, removed buttons
### Files modified:
- [ ] index.html
- [ ] App
- [x] CardContainer
- [x] PokeCard
- [x] PokeForm
- [ ] Login
- [x] FormContainer
- [ ] index.js
- [ ] Other files:

### CODE REVIEW

File: stretch-auth/src/CardContainer/CardContainer.js [hunk 1]
```diff
@@ -5,25 +5,26 @@ import './CardContainer.css'
 
 // needs conditional renders for user & pokemon + user & no pokemon
 const CardContainer = (props) => {
+  const hasTeam = Object.values(props.userPokeTeam).some(poke => !!poke.pokemon)
```
Comment (H-Bo214) [inline stretch-auth/src/CardContainer/CardContainer.js:8]: This line of code is great! Powerful and clean. Nice work.

**Review by H-Bo214 — APPROVED** [review]
Great improvements to the overall look and UX.

File: stretch-auth/src/CardContainer/CardContainer.js (hunk 1 above)
Comment (AurumValian) [reply stretch-auth/src/CardContainer/CardContainer.js:8]: I agree, the double-bang is turning out to be super useful!

File: stretch-auth/src/CardContainer/CardContainer.js [hunk 2]
```diff
@@ -5,25 +5,26 @@ import './CardContainer.css'
...
   return (
     <section className='card-container'>
-      {!props.currentUser.name &&
+      { !props.currentUser.name &&
         <NavLink to='/login' activeClassName='login-message-link'><p className='no-user-message'>Log In Please!</p></NavLink>
       }
-      {props.currentUser.name &&
-        <div className='no-team-display'>
-          <p className='no-team-message'>You Don't Have A Poke Team</p>
-          <Link to='/poke-forms'><button className='no-team-button'>Make a Poke Team!</button></Link>
```
Comment (AurumValian) [inline stretch-auth/src/CardContainer/CardContainer.js:16]: I'm gonna miss the super big button...

**Review by AurumValian — APPROVED** [review]
Looks great!

**Pull request merged**